photon-api/src/main/scala/com/linkedin/photon/ml/util/DefaultIndexMapLoader.scala

```diff
@@ -22,17 +22,19 @@ import org.apache.spark.broadcast.Broadcast
  */
 class DefaultIndexMapLoader(sc: SparkContext, featureNameToIdMap: Map[String, Int]) extends IndexMapLoader {
 
-  @transient
+  @transient // so that _indexMap is not serialized (too big)
```

Comment:
This is borderline in the "noise" comment category. Also, we typically don't do in-line comments.